specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2020-10-01/Clusters.json

```diff
@@ -629,12 +629,6 @@
           "description": "Minimum CapacityReservation value in GB.",
           "type": "integer",
           "format": "int64"
-        },
-        "maxCapacity": {
-          "readOnly": true,
-          "description": "Maximum CapacityReservation value in GB.",
-          "type": "integer",
-          "format": "int64"
         }
```

Comment:
This removing caused model validation failure, you may also change example files.
and can you also send me the PR# which get approval for breaking change?

Reply:
Thanks, fixed. 
Link to PR LA workspace breaking change-> https://github.com/Azure/azure-rest-api-specs/pull/13812

Reply:
Looks the approval is not about "maxCapacity"

Reply:
The property is named a bit different between workspace & cluster resources but operates the same -> limit capacity reservation values. We have introduced new pricing tiers to remove this limit, reduce costs and offer selection for the customer (Enum)

Reply:
@akning-ms , any update? can you advise on how to move on?